### DISCUSSION TITLE
clear command to clear the cli

### DISCUSSION THREAD

**Discussion by iamsachin45** [duckdb/duckdb#15150] [Ideas]
I am thinking if we can provide support for clear the terminal after running "clear" command. Would love to discuss this request further. Also, I would like to work on this feature.

**Comment by Tishj** [comment]
`.clear` sounds like a good idea to me 👍

**Reply by iamsachin45** [reply]
Great, I think I should start working on this. Later we can discuss more. Let me know if you have any questions.

**Reply by maurice1408** [reply]
What's wrong with `.shell clear` or `.shell cls`?
